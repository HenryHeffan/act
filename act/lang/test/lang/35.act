defproc testme (chan?(int<8>) A, B; chan!(int) OUT)
{                                                      
  chan(bool) OUT1;
  chan(bool) tmp;
  int x;
  bool y;
  chp {                                          
    *[ else -> A?x; OUT!int(y) ]
  }                                                    
}                                                      

testme t;
