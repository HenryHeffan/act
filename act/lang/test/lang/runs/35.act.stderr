ERROR: File `35.act', line 8, col 7
	else cannot be the first clause in a selection statement
